ond.py

```diff
@@ -61,59 +63,85 @@ def run_protocol(self, config_):
         )
 
         for test_id in config["test_ids"]:
+            algo_test_params = {
+                "dataset_ids": [],
+                "dataset_root": config["dataset_root"],
+                "image_features": {},
+            }
+
             test_metadata = self.interface.get_test_metadata(
                 session_id=session_id,
                 test_id=test_id
             )
 
-            test_params = {}
-
-            red_light = test_metadata.get("red_light", "")
-
-            # Assume no feedback_params attribute for now.
+            algo_test_data = {}
+            if "red_light" in test_metadata:
+                algo_test_data["red_light_image"] = test_metadata["red_light"]
```

Comment:
Could you use `get` here so that we don't run into key error is `red_light` is not provided in the metadata?

Reply:
Updated in the latest commit to use `get`.